test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -142,6 +154,54 @@ public void ManagePackagesDialogClosesWithDynamo()
 
         }
 
+        [Test]
+        public void PackageManagerClientViewModelTest()
```

Comment:
if you want to stick with a general name can we add a comment describing the test a bit?